modules/gitops-release-notes-1-8-0.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-0_{context}"]
+= Release notes for {gitops-title} 1.8.0
+
+{gitops-title} 1.8.0 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-8-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update,  you can add support for the ApplicationSet Progressive Rollout Strategy feature. Using this feature, you can enhance the ArgoCD ApplicationSet resource to embed a rollout strategy for a progressive application resource update after you modify the ApplicationSet spec or Application templates. When you enable this feature, applications are updated in a declarative order instead of simultaneously. link:https://issues.redhat.com/browse/GITOPS-956[GITOPS-956]
++
+[IMPORTANT]
+====
+ApplicationSet Progressive Rollout Strategy is a Technology Preview feature.
+====
+//https://github.com/argoproj/argo-cd/pull/12103
+
+* With this update, the *Application environments* page in the *Developer* perspective of the {product-title} web console is decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. You do not have to use the `kam` CLI to generate Application Environment manifests for the environments to show up in the *Developer* perspective of the {product-title} web console. You can use your own manifests, but the environments must still be represented by namespaces. In addition, specific labels and annotations are still needed. link:https://issues.redhat.com/browse/GITOPS-1785[GITOPS-1785]
+
+* With this update, the {gitops-title} Operator and the `kam` CLI are now available to use on ARM architecture on {product-title}. link:https://issues.redhat.com/browse/GITOPS-1688[GITOPS-1688]
++
+[IMPORTANT]
+====
+`spec.sso.provider: keycloak` is not yet supported on ARM.
+====
+
+* With this update, you can enable workload monitoring for specific Argo CD instances by setting the `.spec.monitoring.enabled` flag value to `true`. As a result, the Operator creates a `PrometheusRule` object that contains alert rules for each Argo CD component. These alert rules trigger the firing of an alert when the replica count of the corresponding component has drifted from the desired state for a certain amount of time. The Operator will not overwrite the changes made to the `PrometheusRule` object by the users. link:https://issues.redhat.com/browse/GITOPS-2459[GITOPS-2459]
```

Comment:
This part seems a bit off `firing of an alert`. 
Suggestion:
```suggestion
* With this update, you can enable workload monitoring for specific Argo CD instances by setting the `.spec.monitoring.enabled` flag value to `true`. As a result, the Operator creates a `PrometheusRule` object that contains alert rules for each Argo CD component. These alert rules trigger an alert when the replica count of the corresponding component has drifted from the desired count for a certain amount of time. The Operator will not overwrite the changes made to the `PrometheusRule` object by the users. link:https://issues.redhat.com/browse/GITOPS-2459[GITOPS-2459]
```

Reply:
@shipsing agree with removing the word firing. But with the 2nd suggestion, it is the state and NOT the count. So no changes there.